### PR TITLE
Updated App, added landing page and added react-router-dom

### PR DESCRIPTION
**Pull request by TishShaw**
### **Overview**
This pull request introduces the opening screen for the app and integrates React Router DOM for efficient page routing.

### **Changes Made:**

- **Integration of React Router DOM**: Added React Router DOM to enable dynamic page routing throughout the application.

- **Modification to main.jsx**: Implemented BrowserRouter as Router in main.jsx to ensure React Router DOM functionality is accessible app-wide.

- **Update to App.jsx**: Defined the various paths used in the application to facilitate navigation. Enhanced the structure to support the addition of new routes.

- **Creation of Home.jsx**: Developed the opening screen interface.
Included essential elements such as:

     - A prominent title display.
      
     - A 'Start' button, which on activation, navigates the user to the game area.
      
     - A static 'How to' button, intended to provide guidance and instructions for new users

### CODE REVIEW

**Review by LMMSWDesigner — APPROVED** [review]

**Review by KatrinaWright — CHANGES_REQUESTED** [review]
I tested the branch, and it seemed to have lost the styling for the gameboard - please re add this import back into the App.jsx and then it can be merged into main

import './App.css'

**Comment by LMMSWDesigner** [comment]
Return the statement 

import './App.css'

to App.jsx

**Review by LMMSWDesigner — APPROVED** [review]
Fixed css issue

File: reverse-pacman/package.json
Comment (AdventureBear) [inline reverse-pacman/package.json:1]: there is no need to add react router to this project.  Doing so is missing a key feature of reacts ability to conditionally show and hide components.  We want conditional rendering for the start screen, not a new route.

**Review by AdventureBear — CHANGES_REQUESTED** [review]
here is no need to add react router to this project. Doing so is missing a key feature of reacts ability to conditionally show and hide components. We want conditional rendering for the start screen, not a new route.

**Comment by TishShaw** [comment]
I've removed the dependency on react-router-dom, incorporated App.css directly into App.jsx, and used useState hook to elevate state from home.jsx to app.jsx.

File: reverse-pacman/src/main.jsx
Comment (AdventureBear) [inline reverse-pacman/src/main.jsx:1]: this is the only file that still needs to be reverted by removing the Router references.  Everything else looks and works great.

**Review by AdventureBear — CHANGES_REQUESTED** [review]
Everything looks good except main.jsx still has the browser router references. Once those are removed it will be good to merge! Godod job.

**Review by KatrinaWright — APPROVED** [review]
Tested - looks great! 
Thank you for resolving those changes with the app.css and the routing! It looks so cool now!
Merging into main <3

**Review by KatrinaWright — APPROVED** [review]
This is a test to see if I can resolve this request through the GUI as well

**Review by AdventureBear — APPROVED** [review]

**Review by LMMSWDesigner — COMMENTED** [review]
This is fine. It works.

**Pull request merged**